packages/tic-tac-toe/app/controllers/game.ts

```diff
@@ -127,13 +177,37 @@ export default class GameController extends Controller {
       currentChannelState.aOutcomeAddress,
       currentChannelState.bOutcomeAddress
     );
+    if (type === 'victory') {
+      this.transitionToRoute('games');
+    }
+  }
+
+  private async closeChannel(): Promise<void> {
+    await this.tttChannelClient.closeChannel(this.currentGame.getChannelState().channelId);
   }
 
   private updateBoard(player: string, Xs: number, Os: number): void {
     this.player = player;
     this.Xs = Xs;
     this.Os = Os;
   }
+
+  private isWinningMove(marks: number): boolean {
+    return (
+      (marks & TOP_ROW) === TOP_ROW ||
+      (marks & MID_ROW) === MID_ROW ||
+      (marks & BOT_ROW) === BOT_ROW ||
+      (marks & LEFT_COL) === LEFT_COL ||
+      (marks & MID_COL) === MID_COL ||
+      (marks & RIGHT_COL) === RIGHT_COL ||
+      (marks & DH_DIAG) === DH_DIAG ||
+      (marks & UH_DIAG) === UH_DIAG
```

Comment:
nit: if you have a `LINES` constant containing all the lines you can win on, this can be
`LINES.some(line => (line & marks) === line)`

Reply:
I wonder if being more explicit here will be better for readabilty?